library/std/src/sync/mutex.rs

```diff
@@ -219,6 +219,26 @@ impl<T> Mutex<T> {
             data: UnsafeCell::new(t),
         }
     }
+
+    /// Immediately drops the guard, and consequently unlocks the mutex.
+    ///
+    /// To unlock the mutex, one can also call [`std::mem::drop`] on the guard or let the guard go
```

Comment:
`drop` is part of the prelude, so just ``[`drop`]`` might work?

Reply:
Let me try it